aws/table_aws_ecr_image_scan_finding.go

```diff
@@ -163,6 +163,10 @@ func listAwsEcrImageScanFindings(ctx context.Context, d *plugin.QueryData, _ *pl
 			return nil, err
 		}
 
+		if output.ImageScanFindings == nil || output.ImageScanFindings.Findings == nil {
+			return nil, nil
+		}
```

Comment:
```suggestion
		// If the scan is in progress and no findings are available yet, ImageScanFindings is nil
		if output.ImageScanFindings == nil {
			return nil, nil
		}
```
@bigdatasourav @aminvielledebatAtBedrock If we only need to check the first property, `ImageScanFindings`, maybe we can use the code and comment above